### DISCUSSION TITLE
Gradio UI stuck while generating, needs refreshing

### DISCUSSION THREAD

**Discussion by outhipped** [TheLastBen/fast-stable-diffusion#177] [Q&A]
I have to use the gradio server as for some reason I can't use localhost on Chrome. Now when i press generate the notebook shows the operation being run but the UI becomes stalled until I refresh it - losing all settings I indicated before the generation. Is there something I can do to solve this issue? Thank you.

**Comment by TheLastBen** [comment]
Does it show the generated images before stalling ?

**Reply by outhipped** [reply]
sometimes it does but the loading indicator is still stalled. the images appear in my drive.
rarely it unstalls after 20 minutes but I'm usually to eager to generate next batch to wait this long.

**Reply by TheLastBen** [reply]
Why don't you use the local tunnel ?

**Reply by outhipped** [reply]
My chrome can't connect to a local, i'm not sure why but it defaults to a "can't connect" message right away. maybe it's chrome's security precaution to be somehow changed in settings. i will look into it.

**Reply by TheLastBen** [reply]
I think that error message can be easily bypassed

**Reply by outhipped** [reply]
just spent 5 hours trying to fix not being able to connect to http://127.0.0.1:7860/, also tried connecting to it from multiple devices and wifi networks (my primary provider + mobile hotspot). 
is there any way to get the gradio link to work for more than one generation? any help would be appreciated.

**Reply by TheLastBen** [reply]
the link is not  http://127.0.0.1:7860/

**Reply by TheLastBen** [reply]
it should be a long url

**Reply by outhipped** [reply]
Running on local URL:  http://127.0.0.1:7860/ - not workinf
Running on public URL: long gradio link (can provide it privately) - stalling after one or two generations

**Comment by TheLastBen** [comment]
make sure you use the latest colab from the repo

**Reply by outhipped** [reply]
ok will delete sd folder and start fresh, will update thank you.

**Reply by outhipped** [reply]
Thank you. Looks like the localhost that does not work is the one available when I select "use gradio" at the end of the notebook. If i keep it unchecked i get a different local URL (longer one) and this one works. Thank you!

**Reply by TheLastBen** [reply]
happy yo help
